csvcubed/csvcubed/models/cube/qb/catalog.py

```diff
@@ -22,18 +22,24 @@ class CatalogMetadata(CatalogMetadataBase, UriIdentifiable):
     creator_uri: Optional[str] = field(default=None, repr=False)
     publisher_uri: Optional[str] = field(default=None, repr=False)
     landing_page_uris: list[str] = field(default_factory=list, repr=False) 
-    theme_uris: list[str] = field(default_factory=list, repr=False)
-    keywords: list[str] = field(default_factory=list, repr=False)
+    theme_uris: Optional[list[str]] = field(default_factory=list, repr=False)
```

Comment:
Do we need to mark these as `Optional`, won't they automatically be created as an empty list if no value has been provided by the user?